extensions/search-regexp/src/components/CategoriesDropdown.tsx

```diff
@@ -0,0 +1,20 @@
+import { List } from "@raycast/api";
+import { Category } from "../types";
+
+export default function CategoriesDropdown({
+  categories,
+  onCategoryChange,
+}: {
+  categories: Category[];
+  onCategoryChange: (newCategory: string) => void;
```

Comment:
It's always better to create exportable props interface like 
`export interface CategoriesDropdownProps { ... }`